src/main/scala/cr/pulsar/Producer.scala

```diff
@@ -25,37 +25,37 @@ import java.util.concurrent.TimeUnit
 import org.apache.pulsar.client.api.{ MessageId, ProducerBuilder }
 import scala.concurrent.duration.FiniteDuration
 
-trait Publisher[F[_], E] {
+trait Producer[F[_], E] {
 
   /**
     * It publishes a message in a synchronous fashion.
```

Comment:
As we are already doing that, let's get rid of all mentions of the word `publish`